### PR TITLE
Added helper methods for single right delegations in playwright and made changes in cypress tests

### PR DESCRIPTION
**Pull request by acn-Sneha**
## Description
1. Helper methods which can be used for delegations to user and org
2. Made changes in cypress failing tests 
3. Developed few test cases using helper methods of single right delegations. 

## Related Issue(s)
- #898 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ RUN dotnet build src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Altin
 RUN dotnet publish src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.csproj -c Release -o /app_output
 
 # Building the final image
-FROM mcr.microsoft.com/dotnet/aspnet:8.0.5-alpine3.18 AS final
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.18 AS final
```
Comment (acn-Sneha) [inline Dockerfile:18]:
```suggestion
FROM mcr.microsoft.com/dotnet/aspnet:8.0.5-alpine3.18 AS final
```

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.csproj [hunk 2]
```diff
@@ -17,14 +17,14 @@
...
-    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.4" />
     <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.22.0" />
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.20.1" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.6.1" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.5.0" />
```
Comment (acn-Sneha) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.csproj]:
```suggestion
    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.6.1" />
```

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Altinn.AccessManagement.UI.Tests.csproj [hunk 3]
```diff
@@ -8,7 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.5" />
+        <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.4" />
```
Comment (acn-Sneha) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Altinn.AccessManagement.UI.Tests.csproj:11]:
```suggestion
        <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.5" />
```

File: backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.csproj [hunk 4]
```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="JWTCookieAuthentication" Version="4.0.1" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.TagHelpers" Version="2.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.4" />
```
Comment (acn-Sneha) [inline backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.csproj:20]:
```suggestion
    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.5" />
```

File: backend/src/Altinn.AccessManagement.UI/Dockerfile [hunk 5]
```diff
@@ -1,6 +1,6 @@
 #See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
-FROM mcr.microsoft.com/dotnet/aspnet:8.0.5-alpine3.18 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.18 AS base
```
Comment (acn-Sneha) [inline backend/src/Altinn.AccessManagement.UI/Dockerfile:3]:
```suggestion
FROM mcr.microsoft.com/dotnet/aspnet:8.0.5-alpine3.18 AS base
```

File: cypress/package.json [hunk 6]
```diff
@@ -11,7 +11,7 @@
         "lint": "eslint ."
     },
     "devDependencies": {
-        "cypress": "^13.9.0",
+        "cypress": "^13.6.4",
```
Comment (acn-Sneha) [inline cypress/package.json:14]:
```suggestion
        "cypress": "^13.9.0",
```

File: package.json [hunk 7]
```diff
@@ -28,7 +28,7 @@
     "axios": "^1.3.4",
     "classnames": "^2.3.2",
     "cross-env": "^7.0.3",
-    "cypress": "13.8.1",
+    "cypress": "13.6.4",
```
Comment (acn-Sneha) [inline package.json:31]:
```suggestion
    "cypress": "13.8.1",
```

File: package.json [hunk 8]
```diff
@@ -51,19 +51,19 @@
     "cypress-recurse": "^1.35.2",
     "dotenv": "^16.4.5",
     "eslint": "^9.0.0",
-    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-prettier": "^9.0.0",
```
Comment (acn-Sneha) [inline package.json:54]:
```suggestion
    "eslint-config-prettier": "^9.1.0",
```

File: package.json [hunk 9]
```diff
@@ -57,13 +57,13 @@
...
     "eslint-plugin-n": "^17.0.0",
-    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-react": "^7.32.2",
     "husky": "^9.0.0",
     "lint-staged": "^15.0.0",
     "madge": "^7.0.0",
-    "prettier": "^3.2.5",
+    "prettier": "^3.0.0",
```
Comment (acn-Sneha) [inline package.json:66]:
```suggestion
    "prettier": "^3.2.5",
```

File: package.json [hunk 10]
```diff
@@ -57,7 +57,7 @@
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-n": "^17.0.0",
-    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-prettier": "^5.0.0",
```
Comment (acn-Sneha) [inline package.json:60]:
```suggestion
    "eslint-plugin-prettier": "^5.1.3",
```

**Review by allinox — APPROVED** [review]
Nice work 😄
I'm not very familiar with playwright myself, but I trust you've tested that this works, and no tests are failing, so that's very promising!

**Pull request merged**